### ISSUE TITLE
Blinking numlock key when changing between windows

### ISSUE THREAD

**Issue by lenglingxuan** [hyprwm/Hyprland#1751]
My default terminal in hyprland is alacritty. (default means using $Mainmod + Q to open) When I switch between alacritty and other applications, my numlock key blinks(off and on).
After some testing, I found some details:
1. changing between applications beside alacritty won't cause the problem.
2. changing between konsole and other applications won't cause the problem.
3. changing the order and changing from other applications to alacritty won't cause the problem.
4. chaning between alacritty and an empty window won't cause the problem.

**Comment by romanstingler** [comment]
I cannot replicate @lenglingxuan is this still happening on the latest git?

**Comment by Ocyss** [comment]
Yes, I also have this situation. I also use `alacritty`. When the focus is switched, `numlock` will go out. Although pressing the number keys can still be used normally, but the flickering is uncomfortable

**Comment by 0x42697262** [comment]
I thought I have broken my system. This is quite unexpected. I never had this issue before and I don't remember when it happens (maybe during when i switched to master layout?). I also have this problem. Weird.

Running on hyprland 0.33.1-2

**Comment by xcpky** [comment]
Running on 0.40.0-1 
My case is whenever I switch to a new window, I can't use numpad despite the numlock flag is on. I need to press num key twice to use the numpad.

**Comment by romanstingler** [comment]
is this still happening ??

**Comment by this-cat** [comment]
The same situation occurs in version 0.42.0-1.

**Comment by 0x42697262** [comment]
Still happens.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
